src/packer.pkr.hcl

```diff
@@ -80,9 +80,9 @@ locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
-    device_name           = "/dev/xvda"
+    device_name           = "/dev/sda1"
     encrypted             = true
-    volume_size           = 8
+    volume_size           = 200
```

Comment:
When you adjusted the size was this particular value based on anything (final disk usage etc.)? I want to make sure we're being intentional.

Reply:
8 GB seems like it's cutting it a bit close.  I'd be fine bumping the volume up to 256 GB to give ourselves some breathing room.

Reply:
@dav3r updated volume sizes to 256 :+1: